Plugin/Model/PagePlugin.php

```diff
@@ -48,8 +56,22 @@ public function aroundGetContent(Page $subject, callable $proceed)
             return $proceed();
         }
 
-        $this->remotePageContentCache[$remotePageId] = $remotePage['content']['rendered'];
+        $html = $remotePage['content']['rendered'];
+
+        $html = preg_replace_callback("{{(.*)}}", function ($matches) {
+            $match = $matches[0];
+
+            $match = html_entity_decode($match);
+            $match = str_replace("”", "\"", $match); // Opening quote
+            $match = str_replace("″", "\"", $match); // Ending quote
+            $match = str_replace("“", "``", $match); // Double quotes
+            return $match;
+        }, $html);
+
+        $html = $this->filterProvider->getPageFilter()->filter($html);
```

Comment:
Is this necessary? The page already gets filtered by the Page block.

Reply:
I was understanding that this method made sure the individual "shortcodes" got converted into their appropriate widget, after checking this. This is seems not true, and it just requires the appropriate decoding done prior to it.
I will commit my changes.